<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>358</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Phone Numbers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #7cff54; 
	background-color:black;
	height: 40%;
	width: 100%;
	border: 1px solid #7cff54; 
}
QPushButton:hover {
	color: black;
	background-color:#7cff54;
}
QPushButton:disabled {
	color: rgb(72, 148, 49);
	background-color:gray;
	border-color: 1px solid rgb(72, 148, 49); 
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="loadDataButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="saveButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="QTableView" name="phoneNumbers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16000</width>
        <height>20000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
	color:#7cff54;
	font-family: orbitron;
	border: 1px solid #7cff54;
	selection-background-color: #7cff54;
	selection-color: black;
}

QTableView::item:focus {
	selection-background-color: #7cff54;
	selection-color: black;
}

QHeaderView::section{
	color:#7cff54;
	font-family:orbitron;
	border: 1px solid #7cff54;
}</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="removeSelectionButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="newTableButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="closeButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="addRowButton">
      <property name="styleSheet">
       <string notr="true">

</string>
      </property>
      <property name="text">
       <string>Add Row</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
